app/presenters/publishing_api_presenters/detailed_guide.rb

```diff
@@ -83,4 +84,26 @@ def unpublishing_explanation
       Whitehall::GovspeakRenderer.new.govspeak_to_html(item.unpublishing.explanation)
     end
   end
+
+  def national_applicability
+    all_nations = %w(England Northern\ Ireland Scotland Wales)
+    nations = all_nations.reduce({}) { |hash, nation|
```

Comment:
The intention of this block isn't obvious, could it be split into a separate method for readability?